projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -93,6 +93,24 @@ describe('AuthService', () => {
         }
       });
     });
+
+    it('should not set isLoading when service destroyed before checkSession finished', (done) => {
+      (auth0Client.checkSession as jasmine.Spy).and.callFake(() => {
+        return new Promise((resolve) => setTimeout(resolve, 20));
+      });
+
+      const localService = createService();
+
+      localService.ngOnDestroy();
+
+      localService.isLoading$.pipe(bufferTime(25)).subscribe((loading) => {
+        expect(loading.length).toEqual(1);
+        expect(loading).toEqual([true]);
+        done();
+
+        (auth0Client.checkSession as jasmine.Spy).and.resolveTo();
+      });
+    });
```

Comment:
This test is added to fix coverage. This isnt related to the current PR, but as the coverage was complaining I figured we want to solve this for once and for all.